pkg/tenant/profiles.go

```diff
@@ -103,16 +111,9 @@ func (s *Server) ProfileUpdate(c *gin.Context) {
 		return
 	}
 
-	// This endpoint only allows users to update their own profile
-	if req.ID != memberID.String() {
```

Comment:
Removed to make this easier on the frontend since the ID is obtained from the claims.